www/nginx/src/opnsense/scripts/nginx/ngx_functions.js

```diff
@@ -1,13 +1,15 @@
 var fs = require('fs');
 var tls_fingerprints = JSON.parse(fs.readFileSync('/usr/local/etc/nginx/tls_fingerprints.json'));
+// ignore GREASE cipher suite values when compiling a browser fingerprint (see rfc8701)
+const GREASE = ["0x0a0a", "0x1a1a", "0x2a2a", "0x3a3a", "0x4a4a", "0x5a5a", "0x6a6a", "0x7a7a", "0x8a8a", "0x9a9a", "0xaaaa", "0xbaba", "0xcaca", "0xdada", "0xeaea", "0xfafa"];
 
 function check_cipher_array(r, browser_ciphers, fingerprint_ciphers, result) {
-  if (result.status == 'Intercepted') {
+  if (result.status.indexOf('Intercepted') !== -1) {
     return;
   }
   if (browser_ciphers.length > fingerprint_ciphers.length) {
-    // the proxy supports more cipers than the browser -> intercepted
-    result.status = "Intercepted";
+    // the proxy supports more ciphers than the browser -> intercepted
+    result.status = "Intercepted(excess suite)";
```

Comment:
since this lands in a header, It may break other's environment. This needs to be announced carefully.
Brackets in headers is also something to avoid. Some more http header typical structure would be "Intercepted; reason="excess suite" if you look at other headers such as Content-Disposition.